package.json

```diff
@@ -14,8 +14,8 @@
   "license": "GPLv3",
   "dependencies": {
     "ganache-cli": "^6.1.0",
-    "openzeppelin-solidity": "^1.10.0",
-    "truffle": "^4.1.11",
+    "openzeppelin-solidity": "^1.8.0",
+    "truffle": "^4.0.6",
```

Comment:
You might want to specify exact versions here
```
    "openzeppelin-solidity": "1.8.0",
    "truffle": "4.0.6",
```

Reply:
Thanks. I specified exact version of `truffle` and `openzeppelin-solidity`. Also exact version of `web3` on `deploy/package.json` and commited the updated `package-lock.json` files. This should fix the errors you mentioned